packages/gitbook/src/components/DocumentView/OpenAPI/OpenAPI.tsx

```diff
@@ -45,21 +47,28 @@ async function OpenAPIBody(props: BlockProps<DocumentBlockSwagger>) {
         return null;
     }
 
+    const serverUrl = serverUrlCache.get('serverUrl');
+
     return (
-        <OpenAPIOperation
-            data={data}
-            context={{
-                icons: {
-                    chevronDown: <Icon icon="chevron-down" />,
-                    chevronRight: <Icon icon="chevron-right" />,
-                },
-                CodeBlock: PlainCodeBlock,
-                defaultInteractiveOpened: context.mode === 'print',
-                id: block.meta?.id,
-                blockKey: block.key,
-            }}
-            className="openapi-block"
-        />
+        <OpenAPIContext block={block} data={data}>
```

Comment:
Are we not supposed to have a `onUpdate` here?

Reply:
This component is a wrapper around the actual context provider so I could keep it as a client component without polluting the rest of the tree which are server components. I've renamed it and commented as admittedly wasn't very clear.